common/src/main/java/io/homeassistant/companion/android/common/sensors/BluetoothSensorManager.kt

```diff
@@ -133,8 +131,6 @@ class BluetoothSensorManager : SensorManager {
         }
     }
 
-    private val ioScope: CoroutineScope = CoroutineScope(Dispatchers.IO)
```

Comment:
You don't need to remove this one? Good to do it on the IO thread instead of main

Reply:
done